pkg/authentication/handlers.go

```diff
@@ -158,7 +163,16 @@ func (a *API) handleCallback(w http.ResponseWriter, r *http.Request) {
 	a.cookieManager.SetAccessTokenCookie(w, oauth2Token.AccessToken)
 	a.cookieManager.SetRefreshTokenCookie(w, oauth2Token.RefreshToken)
 
-	http.Redirect(w, r, ui.UIPrefix, http.StatusFound)
+	redirectURL, _ := url.ParseRequestURI(ui.UIPrefix)
```

Comment:
Will this work when running under a path? (I think the `base_url` is needed for this)

Reply:
Right, I'll go and add a `ContextPath` spec to handle this. Thank you!